Programs/_testembed.c

```diff
@@ -1182,7 +1182,7 @@ static int test_open_code_hook(void)
     return result;
 }
 
-static int _audit_hook_clear_count = 0;
+static int _audit_hook_clear_count = 0; 
```

Comment:
A trailing space is probably added by mistake.

Reply:
it was added by mistake, i just fixed it!